src/main/java/pl/betoncraft/betonquest/BetonQuest.java

```diff
@@ -17,171 +17,39 @@
-import pl.betoncraft.betonquest.events.VariableEvent;
-import pl.betoncraft.betonquest.events.WeatherEvent;
+import pl.betoncraft.betonquest.database.*;
+import pl.betoncraft.betonquest.events.*;
 import pl.betoncraft.betonquest.item.QuestItemHandler;
-import pl.betoncraft.betonquest.objectives.ActionObjective;
-import pl.betoncraft.betonquest.objectives.ArrowShootObjective;
-import pl.betoncraft.betonquest.objectives.BlockObjective;
-import pl.betoncraft.betonquest.objectives.BreedObjective;
-import pl.betoncraft.betonquest.objectives.ChestPutObjective;
-import pl.betoncraft.betonquest.objectives.ConsumeObjective;
-import pl.betoncraft.betonquest.objectives.CraftingObjective;
-import pl.betoncraft.betonquest.objectives.DelayObjective;
-import pl.betoncraft.betonquest.objectives.DieObjective;
-import pl.betoncraft.betonquest.objectives.EnchantObjective;
-import pl.betoncraft.betonquest.objectives.EntityInteractObjective;
-import pl.betoncraft.betonquest.objectives.ExperienceObjective;
-import pl.betoncraft.betonquest.objectives.FishObjective;
-import pl.betoncraft.betonquest.objectives.KillPlayerObjective;
-import pl.betoncraft.betonquest.objectives.LocationObjective;
-import pl.betoncraft.betonquest.objectives.LogoutObjective;
-import pl.betoncraft.betonquest.objectives.MobKillObjective;
-import pl.betoncraft.betonquest.objectives.PasswordObjective;
-import pl.betoncraft.betonquest.objectives.PotionObjective;
-import pl.betoncraft.betonquest.objectives.RespawnObjective;
-import pl.betoncraft.betonquest.objectives.ShearObjective;
-import pl.betoncraft.betonquest.objectives.SmeltingObjective;
-import pl.betoncraft.betonquest.objectives.StepObjective;
-import pl.betoncraft.betonquest.objectives.TameObjective;
-import pl.betoncraft.betonquest.objectives.VariableObjective;
-import pl.betoncraft.betonquest.objectives.VehicleObjective;
+import pl.betoncraft.betonquest.objectives.*;
 import pl.betoncraft.betonquest.utils.Debug;
 import pl.betoncraft.betonquest.utils.PlayerConverter;
 import pl.betoncraft.betonquest.utils.Updater;
 import pl.betoncraft.betonquest.utils.Utils;
-import pl.betoncraft.betonquest.variables.MathVariable;
-import pl.betoncraft.betonquest.variables.GlobalPointVariable;
-import pl.betoncraft.betonquest.variables.ItemAmountVariable;
-import pl.betoncraft.betonquest.variables.LocationVariable;
-import pl.betoncraft.betonquest.variables.NpcNameVariable;
-import pl.betoncraft.betonquest.variables.ObjectivePropertyVariable;
-import pl.betoncraft.betonquest.variables.PlayerNameVariable;
-import pl.betoncraft.betonquest.variables.PointVariable;
-import pl.betoncraft.betonquest.variables.VersionVariable;
+import pl.betoncraft.betonquest.variables.*;
+
+import java.lang.reflect.InvocationTargetException;
+import java.sql.Connection;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map.Entry;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
So as we already talked on discord all those edited imports need to be reverted to single imports.
I'll do that before merging.